### ISSUE TITLE
Accessibility Role table doesn't work

### ISSUE THREAD

**Issue by rishabh92** [facebook/react-native#40759]
### Description

table present in role below:
https://github.com/facebook/react-native/blob/b7191cde4e36d3d712affc8b714e7d6179642cbb/packages/react-native/Libraries/Components/View/ViewAccessibility.d.ts#L364

When try to use it, it doesn't work as it's missing in AccessibilityRole:
https://github.com/facebook/react-native/blob/b7191cde4e36d3d712affc8b714e7d6179642cbb/packages/react-native/Libraries/Components/View/ViewAccessibility.d.ts#L196

### React Native Version

0.71.13

### Output of `npx react-native info`

npx react-native info
info Fetching system and libraries information...
System:
    OS: Windows 10 10.0.22621
    CPU: (24) x64 12th Gen Intel(R) Core(TM) i9-12900K
    Memory: 104.26 GB / 127.68 GB
  Binaries:
    Node: 14.21.3 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.14.18 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
      Versions: 10.0.22000.0, 10.0.22621.0
  IDEs:
    Android Studio: Not Found
    Visual Studio: 17.7.34031.279 (Visual Studio Enterprise 2022)
  Languages:
    Java: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0
    react-native: 0.71.13 => 0.71.13
    react-native-windows: 0.71.35 => 0.71.35
  npmGlobalPackages:
    react-native: Not Found

### Steps to reproduce

```
<View  accessible={true} role="table">
    <View accessible={true} role="row">
		...
    </View>

    <View accessible={true} role="row">
		...
    </View>

</View>
```

### Snack, screenshot, or link to a repository

N/A

**Comment by NickGerleman** [comment]
@necolas can we find a way to document the roles and other web prop values that no-op?

**Comment by necolas** [comment]
It's already documented https://reactnative.dev/docs/accessibility#role

**Comment by necolas** [comment]
Closing as answered

**Comment by rickhanlonii** [comment]
@necolas I don't see any mention of `table` or other unsupported roles, can we document those to avoid confusion?
